### ISSUE TITLE
LDAP Connection fails on Ubuntu 16.04 LTS PHP 7

### ISSUE THREAD

**Issue by unimatrix73** [adldap/adLDAP#107]
When i try to move from Ubuntu 12.04 with PHP 5.3.10 to Ubuntu 16.04 with PHP 7.0.4 then I get this Error:

ldap_connect(): Could not create session handle: Bad parameter to an ldap routine in ........./adLDAP.php on line 630
ldap_set_option(): supplied argument is not a valid ldap link resource in ..........adLDAP.php on line 634
ldap_set_option(): supplied argument is not a valid ldap link resource in ..........adLDAP.php on line 635

I user PHP LDAP Class (adldap) Version 4.0.4

**Comment by veron-33** [comment]
I think this project and this version (4.0.4) is not maintained anymore. 
You can try actual version of Adldap:
v5: https://github.com/Adldap2/Adldap2/releases/tag/v5.2.17
or newer v6: https://github.com/Adldap2/Adldap2
But I not sure whether it works correctly on PHP7

**Comment by PeterEstephan** [comment]
Ive gotten it to work fine on PHP7 on Ubuntu 16.04. Have you tried: 
apt install php7.0-ldap
service apache2 restart

**Comment by unimatrix73** [comment]
Yes i try this also. But it had not worked. So i move now a while ago to Adldap2 6.1.
